pyproject.toml

```
@@ -57,8 +60,11 @@ coverage = [
     {version = "^7.2", python = ">=3.7"}
 ]
 
-[tool.setuptools_scm]
-write_to = "mox/__version__.py"
+[tool.hatch.version]
+path = "hatch/__about__.py"
```

I believe this should stay as `"mox/__version__.py"` and it will write automatically.

---

The thing is according to docs hatch looks for a variable named `__version__` inside the file pointed py `path` but in `mox/__version__.py` there's already a variable with a different value for the version which is the version of the package and not the version of hatch, hence when i try to build using
` poetry run python -m hatch build` it fails with the message: 
`ValueError: Error getting the version from source `regex`: unable to parse the version from the file: mox/__version__.py
`

---

From the docs, what should be in `__about__.py` is the project version, and not the version of hatch itself. The hatch version is already in `pyproject.toml`. Here is a suggestion so it works with the current `__version__.py` file:
```suggestion
path = "mox/__version__.py"
validate-bump = true
scheme = "semver"
pattern = "__version__ = version = \"(?P<version>[^\"]+)"
```

---

Cool, will patch it and do another PR.